src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -35,11 +37,12 @@ public void loadSomeData() {
 					List<Map> placeCategories = (List<Map>) venue.get("categories");
 					
 					count++;
-					scanner.close();
+					
+					System.out.println("bla bla bla");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)